api/test_lambda_function.py

```diff
@@ -0,0 +1,65 @@
+from lambda_function import lambda_handler, table
+from unittest.mock import patch
+import json
+
+def test_add_stamps_with_no_matching_record_in_DB():
```

Comment:
Is this testing for duplicates?

Reply:
This is testing when there is no matching record in the database. The test after this tests when there is a matching record in the database. In either case, there will never be a duplicate in the database because the DB won't let us add two records with the same partition key and sort key. Therefore, the code also doesn't allow adding of a duplicate record and checks if there is an existing record. If there is an existing record we update that, if not then we create the record